src/tensors/gpu/prod.cpp

```diff
@@ -426,6 +430,44 @@ void CSRProd(marian::Tensor C,
     St_indices = allocator->alloc<int>(numValues);
     St_offsets = allocator->alloc<int>(colsS + 1);
     // transpose the second argument
+#if CUDA_VERSION >= 11000
+    size_t buffer_size;
```

Comment:
Since we cannot test this, but I would like to not lose this contribution, how about we add an ABORT at the end of this block, saying "This code is untested. Please remove this ABORT once tests exist and pass."